components.css

```diff
@@ -750,3 +750,15 @@
     align-items: center;
     margin-left: auto;
 }
+
+.modal{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: auto;
+    background-color: var(--white);
+    justify-content: center;
+    align-items: center;
```

Comment:
If there are multiple places where you're using display:flex, justify-content: center, and align-items:center you can create a utility for that or use grouping selector.

Reply:
I will use them, thanks for the suggestion.